Bugzilla/Mailer.pm

```diff
@@ -198,12 +198,14 @@ sub MessageToMTA {
     my ($part) = @_;
     return if $part->parts > 1;    # Top-level
     my $content_type = $part->content_type || '';
-    $content_type =~ /charset=['"](.+)['"]/;
+    $content_type =~ /charset=(.+)/;
```

Comment:
This is still fragile — there's no guarantee that `charset` will be the last part of the header.

Instead of reinventing the wheel, use [Email::MIME::ContentType](https://metacpan.org/pod/Email::MIME::ContentType) to parse the content-type.

It's currently pulled in by Email::MIME, but be sure to add it as an explicit dependency in case Email::MIME changes.